cmake/readline.cmake

```diff
@@ -187,6 +199,7 @@ MACRO (MYSQL_CHECK_READLINE)
         IF(USE_LIBEDIT_INTERFACE)
```

Comment:
Wouldn't it be simpler to just above `IF(USE_LIBEDIT_INTERFACE OR USE_LIBEDIT_INTERFACE)` and then omit client/mysql.cc and config.h.cmake changes, and omit the `SET(USE_NEW_LIBEDIT_INTERFACE..` below. It looks odd the the C++ implementation in mysql.cc is identicial.

Reply:
Hi Dan,

Your feedback didn't compute 100%, but I think what you meant is now in the updated pull request. I agree this way it's slicker and still works the same way.

If this is not what you meant let me know.

Kind regards,
Seb

Reply:
Yep, that was pretty much it. The `SET(USE_NEW_READLINE_INTERFACE ${LIBEDIT_HAVE_COMPLETION_CHAR})` below isn't used so it could be omitted too.